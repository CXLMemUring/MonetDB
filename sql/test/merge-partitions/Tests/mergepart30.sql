CREATE MERGE TABLE table1 (b varchar(32)) PARTITION BY RANGE ON (b);
CREATE MERGE TABLE table2 (b varchar(32)) PARTITION BY VALUES ON (b);
CREATE TABLE another (b varchar(32));

ALTER TABLE table1 ADD TABLE another; --error, a paritioning clause is required
ALTER TABLE table2 ADD TABLE another; --error, a paritioning clause is required

DROP TABLE another;
DROP TABLE table1;
DROP TABLE table2;
