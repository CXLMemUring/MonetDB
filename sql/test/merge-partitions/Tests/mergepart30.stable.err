stderr of test 'mergepart30` in directory 'sql/test/merge-partitions` itself:


# 09:24:49 >  
# 09:24:49 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-32190" "--port=38020"
# 09:24:49 >  

MAPI  = (monetdb) /var/tmp/mtest-32190/.s.monetdb.38020
QUERY = ALTER TABLE table1 ADD TABLE another; --error, a paritioning clause is required
ERROR = !ALTER TABLE: a range partition is required while adding under a RANGE PARTITION TABLE
CODE  = 42000
MAPI  = (monetdb) /var/tmp/mtest-32190/.s.monetdb.38020
QUERY = ALTER TABLE table2 ADD TABLE another; --error, a paritioning clause is required
ERROR = !ALTER TABLE: a value partition is required while adding for a under a LIST PARTITION TABLE
CODE  = 42000

# 09:24:49 >  
# 09:24:49 >  "Done."
# 09:24:49 >  

