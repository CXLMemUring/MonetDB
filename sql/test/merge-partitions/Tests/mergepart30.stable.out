stdout of test 'mergepart30` in directory 'sql/test/merge-partitions` itself:


# 09:24:49 >  
# 09:24:49 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-32190" "--port=38020"
# 09:24:49 >  

#CREATE MERGE TABLE table1 (b varchar(32)) PARTITION BY RANGE ON (b);
#CREATE MERGE TABLE table2 (b varchar(32)) PARTITION BY VALUES ON (b);
#CREATE TABLE another (b varchar(32));
#DROP TABLE another;
#DROP TABLE table1;
#DROP TABLE table2;

# 09:24:49 >  
# 09:24:49 >  "Done."
# 09:24:49 >  

